requirements/base.in

```diff
@@ -0,0 +1,8 @@
+# Main requirements of your plugin application.
+-c constraints.txt
+
+Django
+djangorestframework
+django-mysql
```

Comment:
is this a requirement ?

Reply:
it was a requirement in the old requirement file

Reply:
I think that yo should add a marker here:

"django-mysql>=3.6; python_version=='3.5'", just like it is currently in setup.py, and the modify setup.py to get the requirements from this file.

Reply:
This pr https://github.com/eduNEXT/eox-tenant/pull/103  removed the django-mysql, so  do we need still here ??

Reply:
I wasn't aware of that. That's great, I guess that we don't need that anymore.

Reply:
i removed the requirement